frontend/tests/playwright/msw/handlers.ts

```diff
@@ -1,8 +1,19 @@
-import { RequestHandler } from 'msw';
+import { RequestHandler, rest } from 'msw';
 
 /**
  * @see https://mswjs.io/docs/basics/request-handler for
  * more information on how to create request handlers for
  * rest and graphql requests.
  */
-export const handlers: RequestHandler[] = [];
+export const handlers: RequestHandler[] = [
+   /**
+    * Prepending the wildcard to the path would capture requests regardless of
+    * origin. Therefore, API requests to Cominor will also be matched against.
+    */
+   rest.get(
+      '*/api/2.0/internal/international_store_promotion/buybox',
```

Comment:
A test could still override this mock easily if we wanted, right?

Reply:
Yes, we can either change it for an entire test runtime https://mswjs.io/docs/api/setup-worker/use#permanent-override or we can override the response for a single request using `res.once`
https://mswjs.io/docs/api/setup-worker/use#permanent-override